further clarifications, addressing comments from previous PR (#15611)
istio/istio.io: content/en/docs/ambient/usage/waypoint/index.md

It is also possible for the waypoint to handle all traffic, only handle traffic sent directly to **workloads** (pods or VMs) in the cluster, or no traffic at all. The types of traffic that will be redirected to the waypoint are determined by the `istio.io/waypoint-for` label on the `Gateway` object.

> Use the `--for` flag on `istioctl waypoint apply` to change the types of traffic that can be redirected to the waypoint:

Reviewer: ```suggestion
Use the `--for` argument to `istioctl waypoint apply` to change the types of traffic that can be redirected to the waypoint:
```

Nit, this is not a flag (a binary choice based on the presence of the parameter). That said, our reference docs use the term 'flag' here.

> | `waypoint-for` value | Original Destination Type |

Reviewer: ```suggestion
| `waypoint-for` value | Original destination type |
```

Author: https://apastyle.apa.org/style-grammar-guidelines/capitalization/sentence-case, today I learned. I always thought title case was correct for column headings

| -------------------- | ------------ |
| `service`            | Kubernetes services |
| `workload`           | Pod IPs or VM IPs |
| `all`                | Both service and workload traffic |
| `none`               | No traffic (useful for testing) |

> Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was **originally** addressed to. If traffic is addressed to a service which does not have a waypoint then waypoint will not be transited even if the eventual workload it reaches does have an attached waypoint.

Reviewer: ```suggestion
Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was _originally addressed to_. If traffic is addressed to a service which does not have a waypoint, a waypoint will not be transited: even if the eventual workload it reaches _does_ have an attached waypoint.
```

> Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was **originally** addressed to. If traffic is addressed to a service which does not have a waypoint then waypoint will not be transitted even if the eventual workload it reaches does have an attached waypoint. 

Reviewer 2: ```suggestion
Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was **originally** addressed to. If traffic is addressed to a service which does not have a waypoint then waypoint will not be transited even if the eventual workload it reaches does have an attached waypoint. 
```

Reviewer 2: (ty)

Author: ty, for some reason my local lsp for markdown (which checks spelling) hates these hugo not quite markdown files...

> Waypoint selection occurs based on the type of destination, `service` or `workload`, which traffic was _originally addressed to_. If traffic is addressed to a service which does not have a waypoint, a waypoint will not be transited: even if the eventual workload it reaches _does_ have an attached waypoint.

Reviewer 3: ```suggestion
Waypoint selection occurs based on the type of destination, service, or workload, which traffic was _originally addressed to_. If traffic is addressed to a service which does not have a waypoint, a waypoint will not be transited: even if the eventual workload it reaches _does_ have an attached waypoint.
```

Author: Oxford comma?

Also, I had added the backticks here to highlight that `service` and `workload` are configurable settings not just generic concepts. Maybe that is incorrect still.

Reviewer 3: Yes, we do oxford commas. And gotcha, I guess I didn't read it that way, but I guess that's what I get for only reading the change and not the text above. I've modified my suggestion that might clarify that a bit more

Author: OK, took another crack and making this read a little cleaner based on, but not exactly adopting your suggestion

Any requests from pods in the ambient mesh to the `reviews-v2` pod IP will now be routed through the `reviews-v2-pod-waypoint` waypoint for L7 processing and policy enforcement.

{{< tip >}}

> The original destination type of the traffic is used to determine if a service or workload waypoint will be used. By using the original destination type the ambient mesh avoids having traffic transit waypoint twice, even if both service and workload have attached waypoints.

Reviewer: We used `service` and `waypoint` (with backticks) above.  Worth being consistent?

Reviewer 3: I don't think they should be backticked. We typically backtick commands and when discussing Kubernetes resources.

Author: I'll second that. In this use service and workload are being used simply to qualify the waypoint. In the previous use I did backtick them because those are possible arguments which can be used with --for which seemed to me like a different use.

This might be better still if I revise the wording to be:

...determine if a service-attached or workload-attached waypoint will...
